ui/src/pages/Users.tsx

```diff
@@ -47,21 +34,28 @@ export const UserShow = () => (
     <SimpleShowLayout>
       <TextField label="ID" source="id" />
       <TextField label="Username" source="username" />
-      <ReferenceArrayField
-        label="Organizations"
-        source="organizations"
+
+      <ReferenceManyField
+        label="Owned Organizations"
```

Comment:
Is there any difference between organizations owned vs. a member of?

Reply:
Yes.. You can delete / modify / send invitations for owned organizations.  We can't easily show member of relations since we would need the [ReferenceManyToManyField](https://marmelab.com/react-admin/ReferenceManyToManyField.html) react component and that's not open source.

Reply:
Got it. Maybe we can do a custom component to get what we need.  (later, though)